modules/end-user-guide/pages/enabling-go-artifact-repositories.adoc

```diff
@@ -0,0 +1,55 @@
+:navtitle: Go
+:keywords: go, artifact-repository, artifact-repositories
+:page-aliases: .:using-go-artifact-repositories
+
+[id="enabling-go-artifact-repositories"]
+= Enabling Go artifact repositories
+
+To enable a Go artifact repository in Go workspaces that run in a restricted environment:
```

Comment:
I believe the IBM SG rule about lead-in sentences ending with a colon doesn't apply in this case because

a) This is (or should be) a context paragraph, not a lead-in sentence.
b) Even if we considered it a lead-in sentence, there's still another section (prerequisites) between the sentence and the actual procedure steps, so using a colon is out of place.

Reply:
Thank you, Robert.
I've rewritten it for all procedures in this PR as a proper sentence that ends with a period.